svgnative/src/SVGDocumentImpl.cpp

```diff
@@ -527,14 +527,13 @@ std::unique_ptr<Path> SVGDocumentImpl::ParseShape(XMLNode* child)
         ry = std::min(ry, height / 2.0f);
 
         auto path = mRenderer->CreatePath();
-        // TODO: Create path for rectangle if rx != ry.
-        if (rx == 0 && ry == 0)
+        if (rx == 0 || ry == 0)
```

Comment:
Last note, we should use `isCloseToZero(rx) || isCloseToZero(ry)` here since it is rendering related.